docs/faang_experiment_metadata.md

```diff
@@ -125,7 +125,7 @@ ChIP-seq DNA-binding proteins experiments should have an `assay type` of  [ChIP-
 
 Required:
 
- * `target protein name` (*ontology term*) the name of the antibody-targeted protein e.g. Histone
+ * `chip target` (*ontology term*) The particular target which the ChIP-seq is aiming for e.g. H3K27me3
```

Comment:
`chip target` (*ontology term*) The target of the ChIP-seq experiment e.g. H3K4Me3, H3K4Me1, H3K27Me3, H3K27Ac, CTCF.

Reply:
addressed in 6cf1f37